lib/upipe-freetype/upipe_freetype.c

```diff
@@ -445,6 +545,131 @@ static int upipe_freetype_set_flow_def(struct upipe *upipe, struct uref *flow_de
     return UBASE_ERR_NONE;
 }
 
+/** @internal @This sets the freetype pixel size.
+ *
+ * @param upipe description structure of the pipe
+ * @param pixel_size pixel size to set
+ * @return an error code
+ */
+static int upipe_freetype_set_pixel_size_real(struct upipe *upipe,
+                                              unsigned pixel_size)
+{
+    struct upipe_freetype *upipe_freetype = upipe_freetype_from_upipe(upipe);
+    upipe_freetype->pixel_size = pixel_size;
+    return UBASE_ERR_NONE;
+}
+
+/** @internal @This gets the x and y minimum and maximum values of the rendered
+ * characters.
+ *
+ * @param upipe description structure of the pipe
+ * @param str a string with the rendered characters
+ * @param bbox_p filled with x and y mininum and maximum value
+ * @return an error code
+ */
+static int upipe_freetype_get_bbox_real(struct upipe *upipe,
```

Comment:
Shouldn't this be called `_upipe_freetype_get_bbox` to keep a consistent coding style with the rest of the codebase?

Reply:
Fixed